release_dashboard/test/fakes/services/fake_conductor.dart

```diff
@@ -19,10 +21,16 @@ import '../fake_start_context.dart';
 /// release dashboard.
 class FakeConductor extends ConductorService {
   FakeConductor({
+    this.fakeCleanContextProvided,
     this.fakeStartContextProvided,
     this.testState,
-  });
+  }) {
+    /// If there is no [fakeCleanContextProvided], initialize a [FakeCleanContext]
+    /// with no exception thrown when [run] is called.
+    fakeCleanContextProvided ??= FakeCleanContext();
+  }
 
+  FakeCleanContext? fakeCleanContextProvided;
```

Comment:
You can mark this as `late FakeCleanContext`

Reply:
Oh right, is `late final` fine?